02-data-types-and-variables/README.md

```diff
@@ -207,3 +207,141 @@ Select debug tab of Visual Studio Code and click `Start Debugging` or click `F5`
 
 </p>
 </details>
+
+## Exercise 2.5: let's make it look more like animal shelter page
+
+Update home page text for it to be more "animal shelter like".
+
+<details>
+<summary>Solution</summary>
+<p>
+
+You need to update `Index.razor` file in folder `Pages`. This is the file that represents our home page. Updated page file should look like this:
+
+```
+@page "/"
+
+<h1>Welcome to our animal shelter page!</h1>
+
+Here you can adopt you future pet.
+
+```
+
+</p>
+</details>
+
+## Exercise 2.6: make animal registration page
+
+Use `Counter` page as a base for the task as it is very similar to the initial version of `Animal Registration` page.
+
+Rename menu item `Counter` to `Animal Registration`.
+
+Update page to count animals, rename `Click me` to `Add animal`, add button `Remove animal` and implemet the logic behind it. 
```

Comment:
Remove "Update page to count animals". Explain what "implement logic behind means".